common/models/scaler_run_info.go

```diff
@@ -1,11 +0,0 @@
-package models
-
-type ScalerRunInfo struct {
-	EnvironmentID           string             `json:"environment_id"`
-	ScaleBeforeRun          int                `json:"scale_before_run"`
-	DesiredScaleAfterRun    int                `json:"desired_scale_after_run"`
-	ActualScaleAfterRun     int                `json:"actual_scale_after_run"`
-	UnusedResourceProviders int                `json:"unused_resource_providers"`
-	PendingResources        []ResourceConsumer `json:"pending_resources"`
-	ResourceProviders       []ResourceProvider `json:"resource_providers"`
-}
```

Comment:
Not having this struct breaks the `printer` package in cli/. Trying to compile the `l0` binary does not work.

Reply:
Aaaah, I understand - I hadn't tried to use the `cli` at all yet. 👍

Reply:
This should be fixed now. ✅

Reply:
The fix should be to remove that code form the `printer` package - not add this struct back in. Is that the case?